pkg/cloud/scope/machine.go

```diff
@@ -35,10 +35,10 @@ import (
 // MachineScopeParams defines the input parameters used to create a new MachineScope.
 type MachineScopeParams struct {
 	AWSClients
-	Client  client.Client
-	Logger  logr.Logger
-	Cluster *clusterv1.Cluster
-	//AWSCluster *infrav1.AWSCluster
+	Client     client.Client
+	Logger     logr.Logger
+	Cluster    *clusterv1.Cluster
+	AWSCluster *infrav1.AWSCluster
 	Machine    *clusterv1.Machine
```

Comment:
Move AWSCluster down? To order them as the MachineScope{ init

Reply:
?

Reply:
As in:
```suggestion
        Client     client.Client
	Logger     logr.Logger
	Cluster    *clusterv1.Cluster
	Machine    *clusterv1.Machine
	AWSCluster *infrav1.AWSCluster
	AWSMachine *infrav1.AWSMachine
```